tests/console/quota.pm

```diff
@@ -35,27 +35,37 @@ sub run {
 
     # install requirements
     zypper_call 'in quota quota-nfs';
+    my $systemd_version = int(script_output('systemctl --version | grep systemd | awk \'{print $2}\''));
+    my $use_templated_service = ($systemd_version >= 256);
+    record_info('Systemd Version', "Detected systemd version: $systemd_version; Use templated service: $use_templated_service");
```

Comment:
`Use templated service:` is always empty

Reply:
It is 1 when is true

Reply:
I didn't write anything about systemd version.